### ISSUE TITLE
Implement 'How to use' section in all the instruments

### ISSUE THREAD

**Issue by yatri1609** [fossasia/pslab-android#972]
**Actual Behaviour**
Right now there are no directions on how to use a particular instrument or a document helping new users implemented in the app.

**Expected Behaviour**
We must have one to help new users get acquainted to the app by themselves or can use it in some confusion.

**Steps to reproduce it**
Open app and open any instrument

**LogCat for the issue**
Not applicable

**Screenshots of the issue**
Not required

**Would you like to work on the issue?**
Yes I would like to implement a help button on the top which can be clicked to refer to the 'How to use' guide.

**Comment by yatri1609** [comment]
@CloudyPadmal Should I implement it and continue with this issue?

**Comment by CloudyPadmal** [comment]
Yes please! :+1:

**Comment by yatri1609** [comment]
@CloudyPadmal All the instruments are not having the common layout as the title bar is missing in a few like that of Multimeter and Oscilloscope so rather than for each instrument can I implement a common guide for all in the main Instrument section itself ?

**Comment by yatri1609** [comment]
@CloudyPadmal Or we can implement the dialogue box in each instrument. Which one will be more effective?

**Comment by yatri1609** [comment]
I have implemented this. Suggest the changes except that of color scheme as I will be doing that later today.

![screenshot_2018-06-08-11-43-16-805_org fossasia pslab](https://user-images.githubusercontent.com/34381723/41142373-1650ea1e-6b13-11e8-8481-6ed68fd65d11.png)
![screenshot_2018-06-08-11-43-23-255_org fossasia pslab](https://user-images.githubusercontent.com/34381723/41142374-1692cb50-6b13-11e8-83c4-42b11fc59485.png)

@CloudyPadmal  Please have a look at this and suggest me the changes and whether I should make a PR for this or not.

**Comment by yatri1609** [comment]
@Vikum94 @harsh-2711 @abhinavraj23 @Avjeet Please have a look at the same and provide suggestions.

**Comment by CloudyPadmal** [comment]
@yatri1609 this is nice! :+1: can you get started?

**Comment by yatri1609** [comment]
Ya I'll implement it for all the buttons.

**Comment by yatri1609** [comment]
Since I'm not finding the in-boarding guide as per our requirement. @CloudyPadmal Should I implement the same with the dialog box?

**Comment by yatri1609** [comment]
![screenshot_2018-06-14-15-27-08-749_org fossasia pslab](https://user-images.githubusercontent.com/34381723/41405511-868bdebe-6fe7-11e8-8990-26d73c82f853.png)

Something like this? I think we can do this for the more important features explaining the users what is it and what does it shows.

**Comment by CloudyPadmal** [comment]
@yatri1609 yes then we can go for an implementation like this! :+1: You can get started!
